stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/components/FileAttachmentImage.kt

```diff
@@ -0,0 +1,46 @@
+package io.getstream.chat.android.compose.ui.attachments.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberImagePainter
+import io.getstream.chat.android.client.models.Attachment
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.chat.android.compose.ui.util.MimeTypeIconProvider
+
+/**
+ * Represents the image that's shown in file attachments. This can be either an image/icon that represents the file type
+ * or a thumbnail in case the file type is an image.
+ *
+ * @param attachment - The attachment we use to show the image.
+ * */
+@Composable
+public fun FileAttachmentImage(attachment: Attachment) {
```

Comment:
Could probably live in one file with `FileAttachmentContent`

Reply:
It's also being used in `MessageInputFileAttachments`, which is why I decoupled it. Made more sense as it was reused